### PR TITLE
problemst:0.1.0

### PR DESCRIPTION
**Pull request by carreter**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A simple and straightforward template for problem sets/homeworks/assignments.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by laurmaedje** [comment]
CI failed because the thumbnail is in the wrong place. It should be one directory higher up.

File: packages/preview/problemst/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+name = "problemst"
+version = "0.1.0"
+entrypoint = "pset.typ"
+authors = ["Willow Carretero Chavez <@carreter>"]
+license = "MIT"
+description = "Simple and easy-to-use template for problem sets/homeworks/assignments." 
+keywords = ["template", "homework", "problem set", "pset", "assignment"]
+categories = ["report"]
+compiler = "0.11.0"
+exclude = ["./template/thumbnail.png"]
```
Comment (laurmaedje) [inline packages/preview/problemst/0.1.0/typst.toml:11]: No need to exclude the thumbnail, that will happen automatically :)

**Comment by carreter** [comment]
> CI failed because the thumbnail is in the wrong place. It should be one directory higher up.

Got it, that must have been why it wasn't being excluded automatically! Should be fixed now.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**